pkg/daemon/daemon.go

```diff
@@ -591,7 +591,7 @@ func (dn *Daemon) completeDrain() error {
 		}
 	}
 
-	if utils.ClusterType == utils.ClusterTypeOpenshift && !dn.openshiftContext.IsHypershift() {
+	if dn.openshiftContext.IsHypershift() && !dn.openshiftContext.IsHypershift() {
```

Comment:
`true` can't be `false`  :smiley: 
```suggestion
	if dn.openshiftContext.IsOpenshiftCluster() && !dn.openshiftContext.IsHypershift() {
```

Reply:
nice catch :+1: